packages/discord.js/typings/index.test-d.ts

```diff
@@ -401,16 +412,14 @@ client.on('messageCreate', async message => {
   const { client, channel } = message;
 
   // https://github.com/discordjs/discord.js/issues/8545
-  {
```

Comment:
Can we disable the rule that is removing the blocks in this file? They're useful for grouping certain tests.